ui/app/routes/vault/cluster/access/methods.js

```diff
@@ -19,6 +19,9 @@ export default class VaultClusterAccessMethodsRoute extends Route {
   };
 
   model() {
+    // when initially mounting and configuring secret engine, ember-data creates a ghost auth config model
+    // we don't want this record appearing in the access list so unload records
+    this.store.unloadAll('auth-method');
```

Comment:
The ghost record was hanging around still, so unloading before `.findAll` removed the duplicate in the list. 
This only happened when an auth method is initially mounted and then immediately configured.
<img width="1058" alt="Screenshot 2024-10-15 at 2 31 08 PM" src="https://github.com/user-attachments/assets/d3d5ac50-511b-4798-aaa1-6a0f808d1079">